composer.json

```diff
@@ -11,7 +11,7 @@
         "illuminate/support": "^5.0",
         "illuminate/filesystem": "^5.0",
         "league/flysystem": "^1.0",
-        "danhunsaker/laravel-flysystem-service": "^1.1"
+        "danhunsaker/laravel-flysystem-service": "dev-master"
```

Comment:
Tagged v1.3 with the changes there, so we probably want to switch to that instead.